packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [54.0.0]
+
+### Changed
+
+- **BREAKING:** Explicitly pass chainIds to detectNfts ([#5448](https://github.com/MetaMask/core/pull/5448))
```

Comment:
```suggestion
- **BREAKING**: The `detectNfts` method in the NFT Detection Controller now accepts chain IDs directly instead of networkClientId, enabling NFT detection across multiple chains simultaneously ([#5448](https://github.com/MetaMask/core/pull/5448))